src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import racingcar.Car;
+import racingcar.Message;
+
+public class InputView {
+
+    public static final String CAR_REGEX = ",";
+    private static final String ONLY_NUMBER_REGEX = "^[0-9]+$";
+
+    public List<String> inputCars() {
+        System.out.println(Message.INPUT_CAR_MESSAGE);
+        String input = Console.readLine();
+        validateInputCars(input);
+        String[] inputCars = input.split(CAR_REGEX);
+        return Arrays.stream(inputCars).collect(Collectors.toList());
```

Comment:
```suggestion
        return Arrays.stream(input.split(CAR_REGEX)).collect(Collectors.toList());
```
한번 밖에 쓰이지 않는 변수를 할당하는 것보다 합치는 것이 어떨까요?

Reply:
놓친거 같아요!  확실히 깔끔해보이네요~